WalletWasabi.Fluent/CrashReport/CrashReporter.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Diagnostics;
+using WalletWasabi.Logging;
+using WalletWasabi.Microservices;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Fluent.CrashReport
+{
+	public static class CrashReporter
+	{
+		public static bool TryInvokeIfRequired(Exception exceptionToReport)
+		{
+			if (exceptionToReport is null)
+			{
+				return false;
+			}
+
+			try
+			{
+				var serializedException = exceptionToReport.ToSerializableException();
+				var base64ExceptionString = SerializableException.ToBase64String(serializedException);
+				var args = $"crashreport -exception=\"{base64ExceptionString}\"";
+
+				var path = Process.GetCurrentProcess().MainModule?.FileName;
+				if (string.IsNullOrEmpty(path))
+				{
+					throw new InvalidOperationException($"Invalid path: '{path}'");
+				}
+
+				ProcessStartInfo startInfo = ProcessStartInfoFactory.Make(path, args);
+				using Process? p = Process.Start(startInfo);
+
+				return true;
+			}
+			catch (Exception ex)
+			{
+				Logger.LogWarning($"There was a problem while invoking crash report: '{ex}'.");
+			}
+
+			return false;
+		}
+
+		public static bool TryGetExceptionFromCliArgs(string[] args, out SerializableException? exception)
+		{
+			exception = null;
+			try
+			{
+				if (args.Length < 2)
+				{
+					return false;
+				}
+
+				if (args[0].Contains("crashreport") && args[1].Contains("-exception="))
+				{
+					var exceptionString = args[1].Split("=", StringSplitOptions.RemoveEmptyEntries)[1].Trim('"');
```

Comment:
Why `StringSplitOptions.RemoveEmptyEntries`? It should split `-exception=something` into exactly 2 items so I would expect:

```suggestion
					var exceptionString = args[1].Split('=', count: 2)[1].Trim('"');
```

Reply:
ACK